packages/v4/patternfly-docs/content/get-started/upgrade.md

```diff
@@ -57,14 +52,26 @@ npx @patternfly/pf-codemods@latest <path to your source code> --exclude menu-sea
 
 7. Run Codemods multiple times to ensure that all issues are flagged and addressed.
 
+# Review and update variable and class names
+
+We changed component class names, layout class names, utility class names, CSS variables, and SCSS variables. Modifier class names have not changed as part of this major release.
 
-# Review and update CSS variables
+These changes mean that any existing CSS overrides will likely be targeting outdated styles. Review your overrides and ensure that they align with our [updated CSS classes and variables.](developer-resources/global-css-variables) 
 
-Because PatternFly 5 brings changes to class and  variable  names, your existing CSS overrides may be targeting outdated styles. Review your overrides and ensure that they align with our [updated CSS classes and variables.](developer-resources/global-css-variables)
+The following table outlines some of the general changes that were made to variable prefixes, but we recommend checking your code against our global css variable guide to cover all changes. 
 
+| Language/style | v4 prefix | v5 prefix|  Note |
+| --- | ---| --- | ---  |
+|  CSS |  `--pf-` |  `$pf-v5-` |
+|  SCSS |  `.pf-`  | `$pf-v5-`  |
+|  SCSS | `@mixin pf-`  | `@mixin pf-v5-`  |
+|  SCSS | `@function pf-`  | `@function pf-v5-`  |
+|  SCSS | `%pf-`  | `%pf-v5-`  |
+| HTML class names  | `.pf-`  |  `.pf-v5-`  | excludes `.pf-m ` |
```

Comment:
I might leave off this last line and make the note more of an asterisk under the table.

Reply:
Do you mean line 70 from the table?

Reply:
sorry - I think I just read this too fast. It's good as is.